client/profile-wizard/steps/store-details/index.js

```diff
@@ -80,9 +80,17 @@ class StoreDetails extends Component {
 			return null;
 		}
 
-		const region = getCurrencyRegion( countryState );
-		const currencyData = getCurrencyData();
-		return currencyData[ region ] || currencyData.US;
+		const Currency = this.context;
+		const country = getCountryCode( countryState );
+		const { currencySymbols = {}, localeInfo = {} } = getSetting(
+			'onboarding',
+			{}
+		);
```

Comment:
Sorry for the fly-by comment, but I'm wondering if you should gather this data from the onboarding store rather than `getSetting` which reads data initialised on the window. Can there be a situation where this data gets updated after initial page load? For instance, if a user edits and goes back?

Reply:
@psealock This data will not get updated after initial load.  It is all the locale info and possible symbols from WC core so this data should not be updated after initial page load.